docs/v1/P4Runtime-Spec.adoc

```diff
@@ -4519,50 +4396,32 @@ control arp_multicast(inout H hdr, inout M smeta) {
     }
   }
 }
-~ End P4Example
+----
 
 At runtime, the client writes the following update in the target (shown in
 Protobuf text format).
 
-~ Begin Prototext
+[source,protobuf]
+----
 type: INSERT
 entity {
   packet_replication_engine_entry {
     multicast_group_entry {
       multicast_group_id: 1
-      replicas {
-        port: "\x05"
-        instance: 1
-      }
-      replicas {
-        port: "\x0c"
-        instance: 2
-        backup_replicas { port: "\x0d" instance: 5 }
-      }
-      replicas {
-        port: "\x12"
-        instance: 3
-        backup_replicas { port: "\x13" instance: 6 }
-        backup_replicas { port: "\x14" instance: 7 }
-      }
-      replicas {
-        port: "\x18"
-        instance: 4
-        backup_replicas { port: "\x19" instance: 8 }
-        backup_replicas { port: "\x1a" instance: 9 }
-        backup_replicas { port: "\x1b" instance: 10 }
-      }
+      replicas { port: "\x05" instance: 1 }
```

Comment:
This looks like an old version of this Protobuf example message.  It should be unchanged from the original.